configure.ac

```diff
@@ -969,13 +969,15 @@
 
     AC_MSG_CHECKING(for libnet.h version 1.1.x)
     if test "$LIBNET_INC_DIR" != ""; then
-        if eval "grep LIBNET_VERSION $LIBNET_INC_DIR/libnet.h | grep -v '1.[[12]]' >/dev/null"; then
-            AC_MSG_RESULT(no)
-            LIBNET_DETECT_FAIL="yes"
-            LIBNET_FAIL_WARN($libnet_dir)
-        else
-            AC_MSG_RESULT(yes)
-        fi
+        LIBNET_VER=`grep LIBNET_VERSION $LIBNET_INC_DIR/libnet.h | sed 's/[[^"]]*"\([[^"]]*\).*/\1/'`
+
+        AS_VERSION_COMPARE([LIBNET_VER], [1.1.0],
```

Comment:
I think it's best if this one is restored to the old check, since this is the generic 'do we have 1.1+' check.